core/Makefile

```diff
@@ -1708,7 +1708,6 @@ $(INTERNAL_OTA_PACKAGE_TARGET): $(BUILT_TARGET_FILES_PACKAGE) $(DISTTOOLS)
 	@echo -e ${CL_YLW}"Package OTA:"${CL_RST}" $@"
```

Comment:
All PRs should be submitted as Gerrit Changes. Please see the documentation at https://wiki.cyanogenmod.org/w/Doc:_using_gerrit